### ISSUE TITLE
[v7.0.1] I think Laragon needs to auto restart when PHP version is changed

### ISSUE THREAD

**Issue by MackSix** [leokhoa/laragon#947]
If I change PHP versions while the services are running, it throws an Entry Point Not Found error message. If I restart all services,   it works. Of course the website doesn't load either. It did this on Laragon v6.0.0 too.

The workaround is to stop everything and restart it. Restarting Apache only, doesn't work.

![entry point not found](https://github.com/user-attachments/assets/9dd37129-e963-4ea9-ae72-071087b5fa3c)

**Comment by x9nico** [comment]
> If I change PHP versions while the services are running, it throws an Entry Point Not Found error message. If I restart all services, it works. Of course the website doesn't load either. It did this on Laragon v6.0.0 too.
> 
> The workaround is to stop everything and restart it. Restarting Apache only, doesn't work.
> 
> ![entry point not found](https://github.com/user-attachments/assets/9dd37129-e963-4ea9-ae72-071087b5fa3c)

Hey, it looks like these issue, no ? => https://github.com/leokhoa/laragon/issues/650

Did you tried to update your Apache httd version ? Like to version 2.4.63 ?

**Comment by MackSix** [comment]
> > If I change PHP versions while the services are running, it throws an Entry Point Not Found error message. If I restart all services, it works. Of course the website doesn't load either. It did this on Laragon v6.0.0 too.
> > The workaround is to stop everything and restart it. Restarting Apache only, doesn't work.
> > ![entry point not found](https://github.com/user-attachments/assets/9dd37129-e963-4ea9-ae72-071087b5fa3c)
> 
> Hey, it looks like this issue, no ? => [#650](https://github.com/leokhoa/laragon/issues/650)
> 
> Did you tried to update your Apache httpd version ? Like to version 2.4.63 ?

No, it is normal for it to do that. You have to restart everything manually.

**Comment by alxndr-w** [comment]
I don't know why but I had this issue with missing extensions with PHP 8.4.2 - it looks like I downloaded the wrong zip. Please go sure that you are using this download page: https://windows.php.net/download#php-8.3 and try it again.

**Comment by leokhoa** [comment]
Fixed in Laragon 8.0.0 https://github.com/leokhoa/laragon/discussions/1058
https://github.com/leokhoa/laragon/releases/tag/8.0.0

Tip: Using PHP NTS would be better as it it is running on different process than Apache.
